pkg/autoscaler/aggregation/aggregation.go

```diff
@@ -38,8 +38,8 @@ type Average struct {
 }
 
 // Accumulate accumulates the values needed to compute an average.
-func (a *Average) Accumulate(_ time.Time, bucket float64Bucket) {
-	a.sum += bucket.sum()
+func (a *Average) Accumulate(_ time.Time, bucket *float64Value) {
+	a.sum += bucket.avg()
```

Comment:
the semantics of this are unusual -- should the `Average.sum` field be renamed?

Reply:
It's a sum of averages.

Reply:
IIUC, before this change, `bucket.sum()` returns the total concurrency of the sample pods in this bucket. After this change, `bucket.avg()` returns the average concurrency per pod in this bucket. Is there an issue for current calculation? If not, this change doesn't sound right to me.

Reply:
I re-did this. It seems to be right now :)